doc/user/cluster_tls.md

```diff
@@ -85,3 +85,30 @@ $ kubectl create secret generic operator-etcd-client-tls --from-file=etcd-ca-crt
 ```
 
 Pass `operator-etcd-client-tls` to `operatorSecret` field.
+
+### Access secure cluster
+
+Since we have set this domain in SAN in etcd server's client cert:
```

Comment:
Assume a secure etcd cluster `example` is up and running.

To access the cluster, use the FQDN `example-client.default.svc.cluster.local`, which matches the SAN of its certificates.